templates/rails_helper.rb

```diff
@@ -1,4 +1,4 @@
-ENV["RACK_ENV"] = "test"
+ENV["RAILS_ENV"] ||= "test"
```

Comment:
As of https://github.com/thoughtbot/suspenders/pull/713 we use `RACK_ENV` only. Same goes for `.travis.yml`.